### DISCUSSION TITLE
Дополнение к меню анимаций.

### DISCUSSION THREAD

**Discussion by HoRRiger** [octothorp-team/public-dobrograd#672] [Предложения]
Это небольшое доловленное к одному из недавних предложений. Было бы прикольно, добавить анимацию лежания на земле, на земле, где руки за головой или же за спиной.  Можно было бы добавить анимацию, где персонаж упереться о, что-то руками, как в GTA5.

**Comment by Ggman77** [comment]
Хорошая идея, добавит чуточку реализма. И механа

**Comment by travka4000** [comment]
я за
